src/slices/scene.tsx

```diff
@@ -0,0 +1,46 @@
+import { createSlice, Dispatch, PayloadAction } from "@reduxjs/toolkit";
+import { RootState } from ".";
+
+interface SceneState {
```

Comment:
should we create a research task to think over and plan out what fields should be kept for the scene? some might stay empty for now and some might get added in the future anyway, but I would prefer if we thought it over rather than modifying every time we need to add a new field

Reply:
let's discuss that during our next meeting.